feature-tests/tests/resources/step-definitions/invoke-api-gateway-happy-path.step.ts

```diff
@@ -0,0 +1,37 @@
+import { defineFeature, loadFeature } from 'jest-cucumber';
+import { generateRandomTestUserId } from '../../../utils/generate-random-test-user-id';
+import { sendSQSEvent } from '../../../utils/send-sqs-message';
+import { invokeApiGateWayLambda } from '../../../utils/invoke-apigateway-lambda';
+
+const feature = loadFeature('tests/resources/features/aisGET/invokeApiGateway-HappyPath.feature');
+
+defineFeature(feature, (test) => {
+  let testUserId: string;
+  let response: any;
+
+  beforeEach(() => {
+    testUserId = generateRandomTestUserId();
+  });
+
+  test('Happy Path - Get Request to /ais/userId - Returns Data for a userId', ({ given, when, then }) => {
+    given(/^I send a request to sqs queue with (.*) data$/, async (aisEventType) => {
+      await sendSQSEvent(testUserId, aisEventType);
+    });
+
+    when(/^I invoke apiGateway to retreive the status of the userId with (.*)$/, async (historyValue) => {
+      response = await invokeApiGateWayLambda(testUserId, historyValue);
+    });
+
+    then(
+      /^I should receive the appropriate (.*), (.*), (.*), (.*) and (.*) for the ais endpoint$/,
+      async (interventionType, blockedState, suspendedState, resetPassword, reproveIdentity) => {
```

Comment:
are we not asserting against the history field in the database?

Reply:
Thanks Chiara, will add this as a separate test with History queryParams etc in a later PR